library.json

```diff
@@ -1,18 +1,16 @@
 {
-  "name": "ConfigManager",
-  "version": "0.8",
-  "keywords": "wifi, wi-fi, config",
-  "description": "ESP8266 WiFi connection manager",
-  "repository":
-  {
-    "type": "git",
-    "url": "https://github.com/nrwiersma/ConfigManager.git"
-  },
-  "frameworks": "arduino",
-  "platforms": "espressif",
-  "dependencies":
-  {
-    "name": "ArduinoJson",
-    "frameworks": "arduino"
-  }
-}
+    "name": "ConfigManager",
```

Comment:
I needed to bump the version number to get PIO to select my version rather than yours. Can you selectively accept from a PR?